Servers/WindowServer/MenuManager.cpp

```diff
@@ -120,8 +120,16 @@ void MenuManager::refresh()
 
 void MenuManager::event(Core::Event& event)
 {
-    if (WindowManager::the().active_window_is_modal())
-        return Core::Object::event(event);
+    auto* active_window = WindowManager::the().active_window();
+    if (active_window && active_window->is_modal()) {
+        if (has_open_menu()) {
```

Comment:
If we use `&& has_open_menu` here we can get rid of a level of indentation 🙂

Reply:
@shannonbooth I thought about that, but I wanted to keep the logical conditions separate to keep make the code clearer. Maybe it's not really a win :).